feat: Merge persoonsgegevensfunction into main

joostvanderborg opened the pull request:
Fixes #Up until now, functionality in Mijn Nijmegen was added in separate
repositories. Because of shared dependencies and frontend, this became
more trouble than it's worth. This commit merges the persoonsgegevens
functionality with the main project.

- dependency updates.

fixes #975

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8367664835

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #966 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

joostvanderborg committed:
feat: Add nowrap style (#967)

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8383476141

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #968 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

joostvanderborg committed:
feat: Merge persoonsgegevensfunction into main (#970)

Up until now, functionality in Mijn Nijmegen was added in separate
repositories. Because of shared dependencies and frontend, this became
more trouble than it's worth. This commit merges the persoonsgegevens
functionality with the main project.

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8397913155

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #977 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8413194625

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #978 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing reviewed (dismissed):
Perssoonsgegevens functie verplaatst naar mijn-nijmegen vanuit mijn-gegevens.

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8428799273

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #980 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8444641842

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #981 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8460429878

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #982 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8475338843

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #983 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8486900046

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #984 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8494371369

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #985 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8501945865

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #986 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8515481122

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #987 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/8531222893

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #988 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing reviewed (approved):
Nogsteeds ok

joostvanderborg committed:
chore: trigger github builds (#989)
